### ISSUE TITLE
Questions about batch size

### ISSUE THREAD

**Issue by isseebx123** [lightvector/KataGo#287]
Thank you for the great experiments and your always kind answers.

Is there any reason to use a batch size of 256? 
I wonder if training becomes unstable when using larger batch sizes, and have you ever experimented with batch sizes?
I am guessing the reason for using the small batch size is because of fixup initialization applied to KataGo.

**Comment by lightvector** [comment]
KataGo has always used batch size 256 because was is the largest power of two that would fit on to a single V100 GPU when training some of the bigger nets. There wasn't enough GPU memory to go larger, it has nothing to do with fixup or not.

**Comment by lightvector** [comment]
It is an good to consider other batch sizes though. I would think that larger is preferable if it would fit within GPU memory. But there's also an interesting question of whether _smaller_ batch sizes would be good too. Very small batch sizes are probably a bad idea when using batch norm because batch norm gets increasingly noisy and inaccurate normalization statistics that will vary much more between train and inference time.

But with fixup, there is no more batch norm, so there should be no limit on how small the batch size can go. To the degree that any particular sample by itself won't change the neural net's inference on any other sample very much, training becomes much more batch-size-independent, I would think.

Was this your thought on how batch sizes and fixup might relate to one another? If so, thanks, that's an interesting thought.
